### PR TITLE
Add prefix to autoscaler_name

### PR DESCRIPTION
**Pull request by rafzei**
Add prefix to autoscaler_name
Change reference to private route table by id instead of name & add as input from awsbi

### CODE REVIEW

**Review by plirglo — DISMISSED** [review]

**Review by mkyc — CHANGES_REQUESTED** [review]
It would be nice to add some documentation regarding changes.

File: resources/defaults.mk [hunk 1]
```diff
@@ -2,6 +2,7 @@ M_NAME ?= epiphany
 M_VPC_ID ?= unset
 M_PUBLIC_SUBNET_ID ?= unset
 M_REGION ?= eu-central-1
+M_PRIVATE_ROUTE_TABLE_ID ?= unset
```
Comment (atsikham) [inline resources/defaults.mk:5]: Is there a way to avoid passing this parameter and use `datasource` with custom `filter` or something similar?

Comment (rafzei) [reply resources/defaults.mk:5]: The second option was to grab it from environment and filter by name but I've talked about it with @sk4zuzu and we agreed we should use it from AwsBI output directly.

**Review by mkyc — APPROVED** [review]
👍

**Review by atsikham — APPROVED** [review]
LGTM

**Pull request merged**